src/channel_internal.c

```diff
@@ -36,8 +39,8 @@ THANDLE_TYPE_DEFINE(CHANNEL_INTERNAL);
 
 typedef struct CHANNEL_OP_TAG
 {
-    PUSH_CALLBACK push_callback;
-    PULL_CALLBACK pull_callback;
+    ON_DATA_CONSUMED_CB on_data_consumed_cb;
+    ON_DATA_AVAILABLE_CB on_data_available_cb;
     void* push_context;
     void* pull_context;
```

Comment:
the variable names for the contexts no longer match the variable names for the callbacks.  Should they?

Reply:
Fixed